.gitignore

```diff
@@ -0,0 +1,7 @@
+.*.swp
+.*.swo
+*.pyc
+*.pyo
+__pycache__/*
+molecule/*/junit.xml
```

Comment:
Would probably be a good idea to add these two lines to `meta_skeleton`

Reply:
https://github.com/oasis-roles/meta_skeleton/pull/4